src/components/toast/toast.stories.mdx

```diff
@@ -1,9 +1,12 @@
+
 import { Meta, ArgsTable, Canvas, Story } from "@storybook/addon-docs";
 import styled from "styled-components";
 
 import Toast from ".";
 import TranslationKeysTable from "../../../.storybook/utils/translation-keys-table";
-import * as stories from "./toast.stories";
+import isChromatic from "../../../.storybook/isChromatic";
+
+export const isOpenForChromatic = isChromatic();
```

Comment:
```suggestion
```
This is not used anywhere in this file, was this addition a mistake or was this part of some other subtask?

Reply:
It was a mistake, however on this PR.